docker-compose.yml

```diff
@@ -26,14 +24,14 @@ services:
     links:
       - ipfs
       - postgres
-      {{#if development}}- ganache{{/if}}
+      - ganache
     environment:
       postgres_host: postgres:5432
       postgres_user: postgres
-      postgres_pass: '{{postgresPassword}}'
+      postgres_pass: 'letmein'
       postgres_db: postgres
       ipfs: ipfs:5001
-      ethereum: {{#if development}}development:http://ganache:8545{{else}}{{env}}:{{ethereumProvider}}{{/if}}
+      ethereum: development:http://ganache:8545
```

Comment:
how about supporting other eth networks ?

Reply:
not really needed or used at the moment.